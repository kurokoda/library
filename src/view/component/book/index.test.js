import React from "react";
import renderer from "react-test-renderer";

import Book from ".";

const props = {
  book: {
    author_name: "John Doe",
    title: "Book Title",
  },
  label: "Label",
  onButtonClick: () => {},
};

it("renders correctly", () => {
  const tree = renderer.create(<Book {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
