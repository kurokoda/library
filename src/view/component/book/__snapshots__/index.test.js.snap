// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="library__book"
>
  <h6>
    Book Title
  </h6>
  <p>
    J
  </p>
  <button
    aria-label={null}
    className="btn btn-secondary"
    onClick={[Function]}
    type="button"
  >
    Label
  </button>
</div>
`;
